src/environment.ts

```diff
@@ -42,5 +42,7 @@ export const splitModeToggleShortcutKey = (isAccelerator = true) =>
   `${cmdOrCtrlShortcutKey(isAccelerator)}+${altKey(isAccelerator)}+S`;
 export const settingsShortcutKey = (isAccelerator = true) =>
   `${cmdOrCtrlShortcutKey(isAccelerator)}+${isMac ? ',' : 'P'}`;
+export const downloadsShortcutKey = (isAccelerator = true) =>
+  `${cmdOrCtrlShortcutKey(isAccelerator)}+${isMac ? ',' : 'J'}`;
```

Comment:
The combination for mac is clashing with the option above. Why can't we go with `J` for all OSes?

Reply:
My bad, it was copy paste from the above line and I forgot to change the logic 🤦‍♂️ I'll change it